include/ola/client/ClientTypes.h

```diff
@@ -29,6 +29,8 @@
 #include <string>
 #include <vector>
 
+#include "common/protocol/Ola.pb.h"
```

Comment:
For now I'll do 

    static OlaInputPort FromProtobufIn(ola::proto::PortInfo port_info);
    static OlaOutputPort FromProtobufOut(ola::proto::PortInfo port_info);